components/request-shape/src/lib.rs

```diff
@@ -76,18 +77,25 @@ fn check_url(req: &IncomingRequest) -> anyhow::Result<()> {
     Ok(())
 }
 
+/// Check that the headers are as expected
 fn check_headers(req: &IncomingRequest) -> anyhow::Result<()> {
-    // Check that the headers are as expected
-    let headers = [
-        ("spin-raw-component-route", "/..."),
+    let expected_headers = [
+        ("spin-raw-component-route", "/:path"),
         ("spin-full-url", "http://example.com/base/path?key=value"),
-        ("spin-path-info", "/path"),
+        ("spin-path-info", ""),
```

Comment:
I'm surprised that this changed to `""`. Where do we document exactly what this value should be equal to?

Reply:
This is the only thing I found - https://developer.fermyon.com/spin/v2/http-trigger#additional-request-information-spin-path-info

Reply:
> The request path relative to the component route (including any base)

Staring at this long enough, I think the `""` is correct.